services/business/services.go

```diff
@@ -74,15 +74,17 @@ func (in *SvcService) castServiceOverview(s *v1.Service, deployments []v1beta1.D
 		Health:       health}
 }
 
-func hasIstioSideCar(deployments []v1beta1.Deployment) bool {
-	for _, deployment := range deployments {
-		if deployment.Spec.Template.Annotations != nil {
-			if _, exists := deployment.Spec.Template.Annotations[config.Get().ExternalServices.Istio.IstioSidecarAnnotation]; exists {
-				return true
-			}
+func hasIstioSideCar(pods []v1.Pod) bool {
+	mPods := models.Pods{}
+	mPods.Parse(pods)
+	hasSideCar := true
+	for _, pod := range mPods {
+		if len(pod.IstioContainers) == 0 {
```

Comment:
So, if at least 1 pod had no side car, we consider the whole service has no side car?
I wonder if we should rename "has istio side car" to "has missing side car", is it seems to be what we're interested to show.

Reply:
You are true, I changed the logic without notice.
It should return true if at least one has sidecar, I think.
Just to maintain same logic it has with the previous function.
I didn't intend to modify that (at least on this PR), good catch.